hive-metastore/src/test/java/org/apache/iceberg/hive/HiveMetastoreTest.java

```diff
@@ -43,8 +43,12 @@ public abstract class HiveMetastoreTest {
   @BeforeClass
   public static void startMetastore() throws Exception {
     HiveMetastoreTest.metastore = new TestHiveMetastore();
-    metastore.start();
-    HiveMetastoreTest.hiveConf = metastore.hiveConf();
+    if (HiveMetastoreTest.hiveConf != null) {
```

Comment:
Trying to understand, how would this happen?

Reply:
As this is happening before class is actually instantiated `HiveMetastoreTest.hiveConf` can be null. I actually saw this in the unit test I added as part of this PR.